department-of-veterans-affairs/caseflow
# Adding disposition to worksheet_issues

Connects #4024 

Part 7! This PR adds disposition to worksheet_issues. 

To test:
1) Connect to your VACOLS instance
2) Run the migration
3) Run the command below (or delete all worksheet issues)
4) Load a worksheet
5) Confirm issues with dispositions are displayed correctly

After this gets merged and deployed, I'll deploy this migration: https://github.com/department-of-veterans-affairs/caseflow/pull/4420

<img width="1065" alt="screen shot 2018-01-19 at 10 27 47 am" src="https://user-images.githubusercontent.com/4306128/35157894-63a2a45c-fd03-11e7-9d22-5547bb482607.png">
